Sources/TZSettingsView/TZSettingsList/TZSettingsSection.swift

```diff
@@ -60,10 +60,14 @@ extension TZSettingsView.Section {
         var color: Color
         /// The font to be used in the themed view or section.
         var font: Font
+        /// The font to be used in the themed view or section.
+        var horizontalPadding: CGFloat?
```

Comment:
as mentioned [here](https://github.com/TrianglZ/TZSettingsView/pull/4/files#r1764250652) change it to non-optional value